log/init.go

```diff
@@ -0,0 +1,69 @@
+package log
+
+import (
+	"github.com/ONSdigital/log.go/v3/config"
+	"github.com/ONSdigital/log.go/v3/log/pretty"
+	"io"
+	"log/slog"
+	"os"
+)
+
+// Initialise is a helper function that creates a new [slog.Logger] with common options and sets it as the default logger
+// for the app. Specifically it adds a namespace attribute to all logs, gets other config from environment variables.
+// It can be configured with extra options passed in as arguments
+func Initialise(ns string, opts ...config.Option) {
+	// Add EnvVar processor and namespace to config options
+	opts = append([]config.Option{config.FromEnv, config.Namespace(ns)}, opts...)
+
+	logger := Logger(opts...)
+
+	SetDefault(logger)
+
+	stdLogger := logger.With(slog.Int("severity", int(INFO))).WithGroup("data")
```

Comment:
Add comment to explain hard-coding of info severity value.

Reply:
Rather than explain the hard-coding, I fixed the issue so it correctly translates the severity (see https://github.com/ONSdigital/log.go/pull/44/commits/881237e1c5483fc904ae175045b18afb77a0b309)